File: 백준/Bronze/1453. 피시방 알바/피시방 알바.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        Integer n = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine());
        HashMap<Integer, Integer> dic = new HashMap<>(100);
        Integer result = 0;
        for (int i = 1; i <= 100; i++) dic.put(i,0);
        for (int i = 0; i < n; i++) {
            Integer val = Integer.parseInt(st.nextToken());
            if (dic.get(val) == 0) dic.put(val,1);
            else result++;
        }
        System.out.println(result);
    }
}

File: 백준/Bronze/1453. 피시방 알바/피시방 알바.py
import sys
input = sys.stdin.readline

n = int(input())
dic = {i:0 for i in range(1,101)}

result = 0
for i in list(map(int,input().split())):
    if not dic[i]: dic[i] += 1
    else: result += 1 

print(result)